src/NBitcoin/Protocol/BitcoinSerializablePayload.cs

```diff
@@ -1,30 +1,22 @@
 ﻿namespace NBitcoin.Protocol
 {
-	public class BitcoinSerializablePayload<T> : Payload where T : IBitcoinSerializable, new()
-	{
-		public BitcoinSerializablePayload()
-		{
+    public class BitcoinSerializablePayload<T> : Payload where T : IBitcoinSerializable, new()
+    {
+        private T @object = new T();
```

Comment:
Can we just have a property that's initialized in the constructor?
Also, please let's have a different name, like "o to the b to the j etc".

Reply:
i don't like it either :-) but being inside these classes i was afraid of changing something too much, but it seems like this is safe, so i've changed it now, it did propagate to quite many files though :/